interface/modules/custom_modules/oe-module-dorn/public/orders.php

```diff
@@ -0,0 +1,147 @@
+            ?>
+        </div>
+    </div>
+    <div class="row"> 
+        <div class="col">
+            <div class="card">
+                <div class="card-body">
+                    <h5 class="card-title"><?php echo xlt("DORN - Lab Orders"); ?></h5>
+                    <div class="row">
+                        <div class="col">
+                            <form method="post" action="orders.php">
+                                <div class="row">
+                                    <div class="col-md-4">
+                                        <select name="form_primaryId">
+                                            <?php foreach ($primaryInfos as $primaryInfo) {
+                                                $selected = $primaryInfo->primaryId === $_GET['form_primaryId'] ? "selected" : "";
+                                                ?>
+                                            <option value='<?php echo attr($primaryInfo->primaryId) ?>' <?php $selected ?>>
+                                                <?php echo xlt($primaryInfo->primaryName) ?>
+
+                                            </option>
+                                            <?php } ?>
+                                        </select>
+                                    </div>
+                                </div>
+                                <div class="row">
+                                    <div class="col-md-4">
+                                        <div class="form-group">
+                                            <label for="form_orderNumber"><?php echo xlt("Order Number") ?>:</label>
+                                            <input type="text" class="form-control" id="form_orderNumber" name="form_orderNumber" value="<?php echo isset($_POST['form_orderNumber']) ? attr($_POST['form_orderNumber']) : '' ?>"/>
+                                        </div>     
+                                    </div>
+                                    <div class="col-md-4">
+                                        <div class="form-group">
+                                            <label for="form_startDateTime"><?php echo xlt("Start Date") ?>:</label>
+                                            <input type="date" class="form-control" id="form_startDateTime" name="form_startDateTime" value="<?php echo isset($_POST['form_startDateTime']) ? attr($_POST['form_startDateTime']) : '' ?>"/>
+                                        </div>
+                                    </div>
+                                    <div class="col-md-4">
+                                        <div class="form-group">
+                                            <label for="form_endDateTime"><?php echo xlt("End Date") ?>:</label>
+                                            <input type="date" class="form-control" id="form_endDateTime" name="form_endDateTime" value="<?php echo isset($_POST['form_endDateTime']) ? attr($_POST['form_endDateTime']) : '' ?>"/>
```

Comment:
rec using the date widgets for dates (see code elsewhere for examples)

Reply:
@claimrevolution I added our datepicker to follow locale settings in Config.
I didn't know how the DORN server requires dates submitted so you'll need to follow up on passed dates to submits.
The DateToYYYYMMDD() function will convert locale date format then you can convert to format needed for DORN from there.